doc/source/development/contributing.rst

```diff
@@ -245,6 +245,10 @@ You'll need to have at least python3.5 installed on your system.
    # Any parent directories should already exist
    python3 -m venv ~/virtualenvs/pandas-dev
    # Activate the virtualenv
+   #    If you are using Windows and Powershell you need to run:
```

Comment:
Are you sure of these? The virtualenv documentation doesn't make mention of these:

https://virtualenv.pypa.io/en/stable/userguide/#activate-script

Generally a little wary of providing instructions like these for third party libraries as our documentation could easily get out of sync. Maybe just link there instead?

Reply:
Ah sorry read the documentation in more detail and see it now. I still think preferable just to link there instead though, as it looks like there could be additional nuances

Reply:
Ah, yes, absolutely correct. I totally forgot about the code execution policies of Powershell.

Is it OK in your opinion, if I write something along the lines: 'If you are using Windows, the activation scripts are located in ~/virtualenvs/pandas-dev/Scripts, for details please refer to: https://virtualenv.pypa.io/en/stable/userguide/#activate-script'